server/darkmoon/settings.py

```diff
@@ -1,17 +1,48 @@
 """Statement imports libraries."""
+
+###########
+# IMPORTS #
+###########
+
+
+from ipaddress import IPv4Address
+
+from dotenv import load_dotenv
 from pydantic import BaseSettings
 
+#############
+# FUNCTIONS #
+#############
+
+load_dotenv()
+
+###########
+# CLASSES #
+###########
+
 
 class Settings(BaseSettings):
     """Settings class."""
 
-    DATABASE_URL: str = "mongodb://darkmoon:password@172.16.5.3/"
+    SCHEME: str
+    IP_ADDRESS: IPv4Address
+    PORT: int
+    USERNAME: str
+    PASSWORD: str
```

Comment:
pydantic type: SecretStr

Reply:
For sensitive information like a password

Reply:
Password was changed from str --> SecretStr and the program is still working. IP_ADDRESS also became IPv4 and everything after "/?" was removed in order to accommodate ruff; however, everything seems to still run (GET and POST both return 200)